dns/prx.org-hosted_zone.yml

```diff
@@ -1561,3 +1561,14 @@ Resources:
           TTL: "300"
           Type: CNAME
           Name: !Sub _9784f5f1de33eb78c3db79157146f598.youthradioleaders.${Domain}
+  Classy:
+    Type: "AWS::Route53::RecordSetGroup"
+    Properties:
+      Comment: Classy
+      HostedZoneId: !Ref HostedZone
+      RecordSets:
+        - ResourceRecords:
+            - vanity.classy.org.
+          TTL: "300"
+          Type: CNAME
+          Name: !Sub support.${Domain}
```

Comment:
Just to confirm, we're assigning all of `support.prx.org` to Classy?

Reply:
That was the intent, but ... now that you ask, it's at least worth considering that support can have many meanings, doubly so for us... Will run it by the (Resource) Development team!

Reply:
OK we're going to go with give instead, thanks for catching that!!